src/System.Net.Http/src/System/Net/Http/HPack/Huffman.cs

```diff
@@ -0,0 +1,426 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace System.Net.Http.HPack
+{
+    internal class Huffman
+    {
+        // TODO: this can be constructed from _decodingTable
```

Comment:
That'll be nice... I'm not going to even try to validate that it's currently correct :)

Reply:
Good :)